website/blog/2022-08-22-narrative-modeling.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Narrative modeling: How structure can tell a story"
 description: "Analytics engineers work in the space between technical and business understanding, but tried-and-true modeling patterns often default to the technical. In this article, Ian Fahey makes a case for a pattern of modeling that follows the narrative of the business it represents."
-slug: narrative-modeling
+slug: narrative-modeling"
```

Comment:
ah there should be no quote here! just `slug: narrative-modeling`

Reply:
Updated that...is the issue that I have so many quotes in the piece? Do I need to escape them somehow?